engine/executor/cvs/cvstask.go

```diff
@@ -159,7 +159,7 @@ func (task *cvsTask) Tick(ctx context.Context) error {
 	// log.Info("cvs task tick", zap.Any(" task id ", string(task.ID())+" -- "+strconv.FormatInt(task.counter, 10)))
 	if task.statusRateLimiter.Allow() {
 		err := task.BaseWorker.UpdateStatus(ctx, task.Status())
-		if errors.ErrWorkerUpdateStatusTryAgain.Equal(err) {
+		if errors.Is(err, errors.ErrWorkerUpdateStatusTryAgain) {
```

Comment:
What is the difference between `errors.Is` and `Equal`?

Reply:
`Equal` will unwarp the error first, and then compare error. Here is an example to illustrate the difference between the two functions.

```go
func TestErrorEqual(t *testing.T) {
	err1 := ErrCreateWorkerTerminate.FastGenByArgs()
	err2 := ErrCreateWorkerTerminate.Wrap(errors.New("invalid config")).FastGenByArgs()
	require.True(t, ErrCreateWorkerTerminate.Equal(err1)) // succeed
	require.True(t, Is(err1, ErrCreateWorkerTerminate))   // succeed
	require.True(t, ErrCreateWorkerTerminate.Equal(err2)) // failed
}
```

Reply:
Thanks. I understand that the core difference is that `Equal` only checks the root cause while `Is` checks the entire err's chain. 

Perhaps we could add some comments to 'reexport.go' or add the example above to illustrate the usage.

Reply:
Developers may not notice comments in 'reexport.go', since they can directly use `ErrXXX.Equal`. I plan to add some limitations to our error definition in later PRs.
An idea to define a new `Error` interface without `Equal` method.
```go
type Error interface {
	RFCCode() RFCErrorCode
	GenWithStackByArgs(args ...interface{}) error
	Wrap(err error) Error
}

var ErrCreateWorkerTerminate Error = errors.Normalize(
		"create worker is terminated",
		errors.RFCCodeText("DFLOW:ErrCreateWorkerTerminate"),
	)
```